docker/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM alpine:3.6 as alpine
```

Comment:
```suggestion
FROM alpine:3.20 as alpine
```

Can you please update all the alpine tag to 3.20 as older version contains some vulnerabilities?